### PR TITLE
Updating packages to use generations

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by Tratcher** [comment]
:shipit:

**Comment by troydai** [comment]
:shipit:

File: src/Microsoft.AspNet.Server.Kestrel.Https/project.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
-﻿{
-    "version": "1.0.0-*",
```
Comment (halter73) [inline src/Microsoft.AspNet.Server.Kestrel.Https/project.json]: Nice that the indentation of our project.jsons are now consistent. :ship:

**Comment by halter73** [comment]
This is already in. https://github.com/aspnet/KestrelHttpServer/commit/dea782163d858b506e700c521455821aed3d5f0c

**Pull request closed without merging**